go.mod

```diff
@@ -4,47 +4,44 @@ go 1.20
 
 require (
 	github.com/NYTimes/gziphandler v1.1.1
-	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
-	github.com/emicklei/go-restful/v3 v3.11.0
-	github.com/go-openapi/jsonreference v0.20.1
-	github.com/go-openapi/swag v0.22.3
-	github.com/golang/protobuf v1.5.2
+	github.com/asaskevich/govalidator v0.0.0-20230301143203-a9d515a09cc2
+	github.com/emicklei/go-restful/v3 v3.12.1
+	github.com/go-openapi/jsonreference v0.21.0
+	github.com/go-openapi/swag v0.23.0
```

Comment:
govalidator, go-restful, jsonreference need review before bumping, these impact kubernetes API surface and behavior